File: app/src/main/java/com/fsoon/android/fsfirsttemplate/net/model/goodpay/ResponseAppVersion.java
package com.fsoon.android.fsfirsttemplate.net.model.goodpay;

import com.fsoon.android.fsfirsttemplate.net.model.ResponseBase;
import com.fsoon.android.fsfirsttemplate.net.model.search.ResponseShopSearchItem;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class ResponseAppVersion extends ResponseBase {

    public ResponseAppVersion(String msg) {
        super(RESPONSE_FAIL, msg);
    }

    public ResponseAppVersion(String code, String msg) {
        super(code, msg);
    }

    @SerializedName("data")
    public ResponseAppVersionData data;

    public class ResponseAppVersionData {
        @SerializedName("appVersion")
        public String appVersion;

        @SerializedName("mandatory")
        public String mandatory;
    }
}

File: app/src/main/kotlin/com/fsoon/android/fsfirsttemplate/net/model/goodpay/ResponseAppVersion.kt
package com.fsoon.android.fsfirsttemplate.net.model.goodpay

import com.fsoon.android.fsfirsttemplate.net.model.ResponseBase
import com.google.gson.annotations.SerializedName

class ResponseAppVersion : ResponseBase {
    constructor(msg: String?) : super(RESPONSE_FAIL, msg) {}
    constructor(code: String?, msg: String?) : super(code, msg) {}

    @kotlin.jvm.JvmField
    @SerializedName("data")
    var data: ResponseAppVersionData? = null

    inner class ResponseAppVersionData {
        @kotlin.jvm.JvmField
        @SerializedName("appVersion")
        var appVersion: String? = null

        @SerializedName("mandatory")
        var mandatory: String? = null
    }
}